### ISSUE TITLE
Return image not found in localhost

### ISSUE THREAD

**Issue by sanPuerquitoProgramador** [toughdeveloper/oc-imageresizer-plugin#12]
Hi, Using 
```
{% for image in post.featured_images|slice(0,1) %}
<a href="{{ post.url }}"><img src="{{ image.path | media | resize(100) }}" alt=""></a>
{% endfor %}
```
Returns always Image not found

Using media, theme or not filter I have the same result.
```
<img src="{{ image.path | media | resize(100) }}" alt="">
<img src="{{ image.path | theme | resize(100) }}" alt="">
<img src="{{ image.path | resize(100) }}" alt="">
<img src="{{ image.path | resize(100,100) }}" alt="">
```

**Comment by toughdeveloper** [comment]
Hey,

If you are using mediafinder (https://octobercms.com/docs/backend/forms#widget-mediafinder)

You should be using the `media` filter. If you try `<img src="{{ image.path | media }}" alt="">` and it doesn't show please let me know.

**Comment by sanPuerquitoProgramador** [comment]
Hi! I'm not using media filter.

`<img src="{{ image.path | media }}" alt="">`

brokes the image:
`/pidgey-local/storage/app/media/pidgey-local/storage/app/uploads/public/ZTV/lNT/ViO/ZTVlNTViOWZjMi5qcGVnITU2ZTRiZGJjMDRmY2MyNTM1MTY1YjJjMmQzYTJjMGY0.jpeg`

If I use without media filter
`<img src="{{ image.path }}" alt="">`

I get the original right image:
`/pidgey-local/storage/app/uploads/public/ZTV/lNT/ViO/ZTVlNTViOWZjMi5qcGVnITU2ZTRiZGJjMDRmY2MyNTM1MTY1YjJjMmQzYTJjMGY0.jpeg`

However, if I use the plugin without filter
`<img src="{{ image.path |resize(200,100)}}" alt="">`

I get the "image not found" path:
`http://localhost/pidgey-local/storage/app/uploads/public/847/474/8b6/thumb__200x100_0_0_crop.jpg`

Thanks!

**Comment by toughdeveloper** [comment]
Is this a public plugin? If so, can you send link please?

The path mentioned looks to already be a resized image, presumably run through the `getThumb()` method in October. 

Also in that case, depending on the attachment type, `{{ image | resize(100) }}` may work.

**Comment by sanPuerquitoProgramador** [comment]
Yes, it's Rain Lab Blog Plugin https://octobercms.com/plugin/rainlab-blog

`{{ image | resize(100) }}` don't work :(

the image object contains: 

```
disk_name: ZTVlNTViOWZjMi5qcGVnITU2ZTRiZGJjMDRmY2MyNTM1MTY1YjJjMmQzYTJjMGY0.jpeg,
file_name: e5e55b9fc2.jpeg,
file_size: 69742,
content_type: image//jpeg,
title: null,
description: null,
field: featured_images,
sort_order: 1081,
created_at: 2016-12-29 17:21:29,
updated_at: 2016-12-29 17:21:29,
path: //pidgey-local//storage//app//uploads//public//ZTV//lNT//ViO//ZTVlNTViOWZjMi5qcGVnITU2ZTRiZGJjMDRmY2MyNTM1MTY1YjJjMmQzYTJjMGY0.jpeg,
extension: jpeg
```

**Comment by sanPuerquitoProgramador** [comment]
I found the issue!!
If the October site is in a subfolder, the $file->path in Image class::construct function fails, because file_exists($filePath) returns FALSE

```
 $this->filePath = (file_exists($filePath))
            ? $filePath
            : base_path() . DIRECTORY_SEPARATOR . $this->parseFileName($filePath);
```
Here is why:
My October site is in http://localhost/pidgey-local/ so, the image path is: 

`/pidgey-local/storage/app/uploads/public/ZDY/2NG/EyM/ZDY2NGEyM2I1YS5qcGVnIWU2MzE0ZDAxMDI0NDNiZWU4ZmJkMGU2NjM0MmFiMTMx.jpeg`

In this case, the plugin can not found the image and returns the not found image

If I use a replace filter 

`<img src="{{ image.path | replace({'/pidgey-local/': ''}) | resize(120,120)  }}" alt="">`

the image path will be: 

`storage/app/uploads/public/ZDY/2NG/EyM/ZDY2NGEyM2I1YS5qcGVnIWU2MzE0ZDAxMDI0NDNiZWU4ZmJkMGU2NjM0MmFiMTMx.jpeg`

and works like a charm 😄 

I had this same issue in a plugin I recently developed, so, in my plugin, I ask to the user if the OctoberCMS site was installed in a subfolder, if yes, I take string and use inside the path to the file.

Here is the configuration screen of my plugin

https://octobercms.com/storage/app/uploads/public/586/f32/e68/586f32e68f928884286052.png

And, if you want to see the complete code, here is the link to the plugin

https://octobercms.com/plugin/pollozen-joomlacsvimport

So, I see three solutions here:

1. In the plugin, found a solution for locate the image with or without a subfolder.
2. In the plugin configuration, ask to the user for a subfolder installation, as I did in my plugin.
3. In the documentation, specify this case can be skipped using a filter in the image path
`<img src="{{ image.path | replace({'/pidgey-local/': ''}) | resize(120,120)  }}" alt="">`

If you find the first point, I will replicate it to my Joomla-October plugin 🤓  because I was looking the same solution but there was not enough coffee... XD

Good day!

**Comment by toughdeveloper** [comment]
@sanPuerquitoProgramador - Ah, makes sense! I will have a look ASAP and get a fix together. 

Thanks for taking the time to post details.

**Comment by toughdeveloper** [comment]
Hey @sanPuerquitoProgramador .

Sorry the for the delay. This issue should now be fixed as of version 1.3.1. (https://github.com/toughdeveloper/oc-imageresizer-plugin/commit/dc6486d1d5f841806b5f814807adc96fd868fa82) 

If you have any further issues, please let me know. 

Thanks,
Matt
